### PR TITLE
Bug/529 bug inconsistency from setup to setup

### LINKED ISSUE #529: Bug: Inconsistency from setup to setup
## Problem Description

1. Running `poe gui` works on one PC, but not any others. On most PCs, the Interlocking can not set the second spawn fahrstrasse, because is is not free.
3. It should run on every PC.
4. We can't ship this one PC.

## Steps For Reproduction

1. run `poe gui`

### PR DESCRIPTION
**Pull request by Lietze**
Fixes #529 

The gui should run now (with the changes to `run`

## PR checklist

- [ ] Acceptance criteria fulfilled
- [ ] Additional features are tested
- [ ] Docs (code, wiki & diagrams) updated
- [ ] Breaking changes anounced
- [ ] Added classes that inherit from `BaseModel` to `src.constants.tables`
- [ ] Dev-branch has been merged into local branch to resolve conflicts
- [ ] Tests and linter have passed AFTER local merge
- [ ] Another dev reviewed and approved

### CODE REVIEW

**Review by antonykamp — APPROVED** [review]
Works on my machine 🥳

File: src/wrapper/simulation_object_updating_component.py [hunk 1]
```diff
@@ -153,6 +154,7 @@ def _remove_stale_vehicles(self):
     def _fetch_initial_simulation_objects(self):
         folder = path.dirname(self._sumo_configuration)
         inputs = next(sumolib.xml.parse(self._sumo_configuration, "input"))
+        print(inputs)
```
Comment (LucasDerReisende) [inline src/wrapper/simulation_object_updating_component.py]: Should this statement be removed?

**Review by LucasDerReisende — COMMENTED** [review]
Looks good, works on my machine. Please have a look at my comment :)

**Review by instant-sky — APPROVED** [review]
Works on my machine too! Thank you for fixing that! 🎰

File: src/wrapper/simulation_object_updating_component.py (hunk 1 above)
Comment (Lietze) [reply src/wrapper/simulation_object_updating_component.py]: yes I will do that. Good chatch

**Pull request merged**